src/EnergyPlus/Vectorize.hh

```diff
@@ -0,0 +1,34 @@
+#ifndef EnergyPlus_vectorize_hh_INCLUDED
+#define EnergyPlus_vectorize_hh_INCLUDED
+
+// EnergyPlus Vectorization Support
+
+// ObjexxFCL Headers
+#include <ObjexxFCL/vectorize.hh> // ASSUME_ALIGNED and RESTRICT macros
+
+// C++ Headers
+#include <cassert>
+#include <cstdint>
+
+// EnergyPlus headers
+#include "EnergyPlus.hh"
+
+namespace EnergyPlus {
+
+	int const VEC_LENGTH = VEC_ALIGN / sizeof(Real64);
+
+#define ROUND_TO_VEC_LENGTH(I) (((I) + ((VEC_LENGTH) - 1)) & ~((VEC_LENGTH) - 1))
+
+
+#define EXPLICIT_VECTORIZATION
+
+	// EXPLICIT_VECTORIZATION assumes that vectors are aligned,
+	// which is not a valid assumption on 32-bit builds
+#if defined(_WIN32) && !defined(_WIN64)
```

Comment:
@DeadParrot I highly suggest NOT targeting AVX/2 for default EnergyPlus. I think if people compile EnergyPlus on their own hardware that supports AVX then, yes, they can take advantage of the bigger pipeline.

I say not AVX/2 for default setting because the earliest processor for AVX is Sandy Bridge (2011) and AVX2 is Haswell (2013). I know my two CI machines have older processors than these.